### DISCUSSION TITLE
add option to schedule automatic installation of updates

### DISCUSSION THREAD

**Discussion by YuriMri** [Devolutions/UniGetUI#2721] [General]
It would be great if UniGetUI had the option to set up installation of updates to certain time.
I see the option to "Install available updates automatically" and I presume this will run on next update check interval.
Option to schedule daily/weekly/monthly to 13:00 (or any time) by adding a file to %localappdata%\UniGetUI with specified syntax would be a super simplification to some admins.

**Comment by marticliment** [comment]
This is a duplicate feature request

**Comment by Lockszmith-GH** [comment]
A simple workaround would be to make UniGetUI check and install updated automatically on startup, and have Task Scheduler initiate at a specific time.

`DisableAutoCheckforUpdates` is the [setting file](https://github.com/marticliment/UniGetUI/discussions/2718#discussioncomment-10608472) you can add if you want to start UniGetUI without the auto checking mechanism.
